_episodes/12-cmdline.md

```diff
@@ -873,13 +873,17 @@ the program now does everything we set out to do.
 > >
 > > def main():
 > >     script = sys.argv[0]
-> >     action = sys.argv[1]
-> >     if action not in ['--min', '--mean', '--max']: # if no action given
-> >         action = '--mean'    # set a default action, that being mean
-> >         filenames = sys.argv[1:] # start the filenames one place earlier in the argv list
+> >     if len(sys.argv) == 1: # there are no args but there may be a pipe
```

Comment:
Small style suggestion: changing to use two spaces before `#` and sticking to that throughout the code would help it look more consistent (and adhere to PEP-8). (I realize the spaces were inconsistent before you worked on this :).)